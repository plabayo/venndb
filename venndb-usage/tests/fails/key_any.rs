use venndb::VennDB;

#[derive(Debug, VennDB)]
struct Employee {
    #[venndb(key, any)]
    country: String,
}

fn main() {}
