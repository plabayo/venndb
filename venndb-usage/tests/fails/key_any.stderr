error: Invalid field-level `venndb` attribute
       Cannot have both `key` and `any`
 --> tests/fails/key_any.rs:5:19
  |
5 |     #[venndb(key, any)]
  |                   ^^^
